src/modules/universal-discovery/components/content-creator/content.creator.component.js

```diff
@@ -24,7 +24,20 @@ export default class ContentCreatorComponent extends Component {
 
     handlePublish() {
         this._refIframe.contentWindow.onbeforeunload = () => {};
-        this._refIframe.contentWindow.document.body.querySelector('#ezrepoforms_content_edit_publish').click();
+        this.submitForm();
+    }
+
+    submitForm() {
+        const selectors = ['#ezrepoforms_content_edit_publish', '#ezrepoforms_user_create_create'];
+
+        for(let selector of selectors) {
+            let submit = this._refIframe.contentWindow.document.body.querySelector(selector);
+            if(submit) {
+                submit.click();
+
+                return;
```

Comment:
Did you try this in the UI?

Reply:
Nevertheless, this is not what I suggested. We should not rely on "if exists" but rather have some special selector as I proposed in the previous comment.

Reply:
"if exists" is always required if we want to avoid errors in the console. I really do not see the difference, 
we have a special selector even two. This is not a script for universal purpose but rather a special one that supports the process of creating content on the fly. In this procees, we support 2 forms only. I my opinion is better to list those forms explicitly as is done currently.

Reply:
For maintenance purposes. If you add one common selector you will no need to come into this JS when you want to add support for nex form.

In my case, it's not like a charm :)
<img width="419" alt="Screenshot at Apr 11 10-37-29" src="https://user-images.githubusercontent.com/12594013/55943087-2a615a00-5c46-11e9-97fc-89c89dd8ffbe.png">

Reply:
Your code is a bit different

Reply:
Yes, it is. What I meant it should be `break;` not `return;`, empty `return;` at the end at the method is really new for me.

Reply:
Changed, new implementation is totally maintenance-free.